base/jobs/docker/1.0/py3/requirements.txt

```diff
@@ -1,16 +1,16 @@
-amazon-braket-default-simulator==1.26.0
-amazon-braket-pennylane-plugin==1.28.0
-amazon-braket-schemas==1.22.0
-amazon-braket-sdk==1.87.0
-awscli==1.34.22
-botocore==1.35.22
-boto3==1.35.22
+amazon-braket-default-simulator==1.26.1
+amazon-braket-pennylane-plugin==1.30.2
+amazon-braket-schemas==1.22.3
+amazon-braket-sdk==1.88.2.post0
+awscli==1.36.13
+botocore==1.35.72
+boto3==1.35.72
```

Comment:
Should these be updated in BDK too?

Reply:
Probably yes, dependabot will do that there as well.